tests/RobotFramework/tests/tedge/http_file_transfer_api.robot

```diff
@@ -27,6 +27,15 @@ Get Put Delete
     Should Be Equal    ${get}    test of put
     Execute Command    curl -X DELETE http://${DEVICE_IP}:${PORT}/tedge/file-transfer/file_a
 
+File transfer using tedge cli
+    Setup    skip_bootstrap=False
+
+    Execute Command    tedge http put /tedge/file-transfer/file_b "content to be transfered"
+    ${content}=    Execute Command    tedge http get /tedge/file-transfer/file_b
+    Should Be Equal    ${content}    content to be transfered\n
+    Execute Command    tedge http delete /tedge/file-transfer/file_b
+    Execute Command    tedge http get /tedge/file-transfer/file_b    exp_exit_code=1
```

Comment:
Not necessarily an action item, but if you don't care about the exact exit code, and you just want to check that it returns a non-zero exit code, then you can use (`!0`). for example:

```
    Execute Command    tedge http get /tedge/file-transfer/file_b    exp_exit_code=!0
```